### ISSUE TITLE
[BUG] Crash on setup step 2/6 (NullPointerException generating toast?)

### ISSUE THREAD

**Issue by e-t-l** [Mahmud0808/Iconify#621]
### Describe the bug

Granted superuser and storage permissions yet crashes during setup.

### Steps to reproduce

1. Grsnt all requested permissions
2. Tap "Get Started"

### Expected behavior

Setup succeeds.

### Screenshots

Not a screenshot but a crash report:
```
type: crash
osVersion: google/husky/husky:14/AP2A.240805.005/2024082200:user/release-keys
package: com.drdisagree.iconify:17
process: com.drdisagree.iconify
processUptime: 6098 + 287 ms
installer: com.machiav3lli.fdroid

java.lang.NullPointerException: Can't toast on a thread that has not called Looper.prepare()
	at com.android.internal.util.Preconditions.checkNotNull(Preconditions.java:168)
	at android.widget.Toast.getLooper(Toast.java:186)
	at android.widget.Toast.<init>(Toast.java:171)
	at android.widget.Toast.makeText(Toast.java:502)
	at android.widget.Toast.makeText(Toast.java:491)
	at dy.b(Unknown Source:1738)
	at hJ.run(Unknown Source:54)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1012)
```

### Device information

- Device: Google Pixel 8 Pro
- OS: GrapheneOS 14 QPR2 **compatibility mode enabled, memory hardening disabled
- App version: 6.7.0
- Magisk: Magisk Delta fork R6687BB53-kitsune (27001)

**Comment by Mahmud0808** [comment]
Read the `App Changes` part from changelog

**Comment by e-t-l** [comment]
> 
>   -  Bump AGP and dependencies
>   -  Updated translations (thanks to all the contributors on Crowdin)
>   -  And many more improvements...
> 
> Also included in this version are bug fixes and other improvements.

Thanks for responding quickly, but this doesn't contain any information related to my issue?

**Comment by DHD2280** [comment]
> > * Bump AGP and dependencies
> > * Updated translations (thanks to all the contributors on Crowdin)
> > * And many more improvements...
> > 
> > Also included in this version are bug fixes and other improvements.
> 
> Thanks for responding quickly, but this doesn't contain any information related to my issue?

You are reporting an issue on version 6.7.0
Yesterday was released 6.9.0.

**Comment by Mahmud0808** [comment]
> You are reporting an issue on version 6.7.0 Yesterday was released 6.9.0.

Oh wait didn't even notice that. What am I gonna do with reports from an old version? Makes no sense

**Comment by e-t-l** [comment]
Apologies, I just saw the version mismatch too. v6.7.0 is the latest version available in F-Droid just so you know.

I tried installing v6.9.0 from github but got an installation error - SAI says it's missing AndroidManifest.xml, which seems weird. Are you sure the v6.9.0 apk isn't corrupted?

**Comment by DHD2280** [comment]
> Apologies, I just saw the version mismatch too. v6.7.0 is the latest version available in F-Droid just so you know.
> 
> I tried installing v6.9.0 from github but got an installation error - SAI says it's missing AndroidManifest.xml, which seems weird. Are you sure the v6.9.0 apk isn't corrupted?

Absolutely, the build is done with GitHub actions.

**Comment by Mahmud0808** [comment]
> Apologies, I just saw the version mismatch too. v6.7.0 is the latest version available in F-Droid just so you know.
> 
> I tried installing v6.9.0 from github but got an installation error - SAI says it's missing AndroidManifest.xml, which seems weird. Are you sure the v6.9.0 apk isn't corrupted?

You do not need SAI. Iconify isn't a split apk.

**Comment by e-t-l** [comment]
Ok, I think the first time i downloaded v6.9.0 it must have just been a corrupted download. I downloaded the apk again and this time it installed fine...

Although when I try to open Iconify now, I get the initial "Get Started" setup page. If I tap the Get Started button, though, Iconify crashes with another NullPointerException (logs pasted below). It's worth noting that I still have my Iconified system customizations working; I just can't open the app.

```
type: crash
osVersion: google/husky/husky:14/AP2A.240805.005/2024082200:user/release-keys
package: com.drdisagree.iconify:21
process: com.drdisagree.iconify
processUptime: 5566 + 321 ms
installer: com.android.packageinstaller

java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.Object.equals(java.lang.Object)' on a null object reference
	at F10.c(Unknown Source:2)
	at com.drdisagree.iconify.ui.views.OnboardingView.a(Unknown Source:1545)
	at Uz.h(Unknown Source:11)
	at M5.i(Unknown Source:4)
	at mh.run(Unknown Source:111)
	at S0.run(Unknown Source:788)
	at GL.run(Unknown Source:2)
	at sd.run(Unknown Source:91)
	Suppressed: zg: [yJ{Cancelling}@4052246, Dispatchers.IO]
```

**Comment by e-t-l** [comment]
(Unrelatedly, when I tried to create a magisk module for the custom volume panel, Magisk "failed to patch SystemUI." No idea what that's about)

**Comment by Mahmud0808** [comment]
You must clear data, please read change log. And volume module template developer isn't much active anymore. So can't help about it either.

**Comment by e-t-l** [comment]
I understand now, thanks for your assistance

**Comment by e-t-l** [comment]
If you don't mind helping me with one last question, I tried installing the LSposed module, and I followed the directions exactly from your app, but when I restart systemui, the screen immediately goes black and will not turn back on. As far as I can tell, the rest of the phone is still on and operational, just the screen no longer works.

If I reboot the device, it fails to boot* and I have to safeboot and disable/uninstall LSposed before I can boot normally again. Iconify is the only module installed in LSposed.

(*the failure to boot is strange. It's not exactly a boot _loop_, but it goes through the normal boot sequence until it gets to the splash screen that says "Pixel is starting..." and then it gets stuck on that screen. While it's true that my phone is a Google Pixel 8 Pro, my _ROM_ is pure AOSP, with no google pixel OS bits whatsoever. That "Pixel is starting..." splash screen is NOT part of my normal boot sequence. It _only_ appears when the bootup gets stuck like this.)
